firmware/src/network_model.c

```diff
@@ -333,14 +290,11 @@ int NetworkModel_SetDHCPMode(const RDMHeader *header,
 
 int NetworkModel_GetZeroconfMode(const RDMHeader *header,
                                  const uint8_t *param_data) {
-  unsigned int index = 0;
```

Comment:
Can you have it more like this to simplify further and remove the duplicate Nack code too?